xbmc/platform/darwin/ios-common/DarwinEmbedNowPlayingInfoManager.mm

```diff
@@ -46,6 +58,12 @@ - (void)onPlay:(NSDictionary*)item
     dict[MPMediaItemPropertyGenre] = [genres componentsJoinedByString:@" "];
 
   NSString* thumb = item[@"thumb"];
+  NSString* epg_icon = item[@"epg_icon"];
+  if (epg_icon && epg_icon.length > 0)
+  {
+    // If we have the TV show, icon use it instead of the channel logo.
+    thumb = epg_icon;
+  }
```

Comment:
If we do the previous comment, there would be no need for this block, as thumb will contain the relevant icon info